.github/workflows/continuous-deployment.yml

```diff
@@ -1,23 +1,32 @@
 name: Continuous deployment
-run-name: Continuous deployment of ${{ github.ref }} to "copilotmigration"
+run-name: Continuous deployment of ${{ github.ref_name }} to QA and Test
 
 on:
   push:
-    branches: [main]
+    branches: [MAV-921_continuous_deployment_job]
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: "Deployment environment"
```

Comment:
```suggestion
        description: Deployment environment
```

Reply:
After the last change, their purposes are more separated:

`continuous-deployment.yml` is now purely for continuous deployment to QA and test and cannot be run manually anymore.

`deploy-mavis.yml` is intended to be triggered manually. This is the standard workflow to perform a manual deployment to any environment, including production